shell/platform/android/test/io/flutter/plugin/platform/PlatformPluginTest.java

```diff
@@ -52,10 +77,12 @@ public void platformPlugin_hasStrings() {
     PlatformChannel fakePlatformChannel = mock(PlatformChannel.class);
     PlatformPlugin platformPlugin = new PlatformPlugin(fakeActivity, fakePlatformChannel);
 
-    clipboardManager.setText("iamastring");
+    ClipData clip = ClipData.newPlainText("label", "Text");
+    clipboardManager.setPrimaryClip(clip);
     assertTrue(platformPlugin.mPlatformMessageHandler.clipboardHasStrings());
 
-    clipboardManager.setText("");
```

Comment:
Actually - why doesn't this work anymore?

Reply:
Thank you for your quick reply.
setText() is deprecated